### DISCUSSION TITLE
doris2.0.15版本简单主键查询比1.2.2版本慢好多，对比profile,时间都多在fe阶段

### DISCUSSION THREAD

**Discussion by bestpayAmes** [apache/doris#43151] [A - General / Q&A]
![image](https://github.com/user-attachments/assets/243190ff-ea2c-4321-b497-558cd9761328)
![image](https://github.com/user-attachments/assets/c8cba1a6-67a8-4f0b-afd8-a8e667a3fc4b)
sql：select * from tbl_point_query where key='XXX';
关闭了enable_nereids_planner，enable_pipeline_engine变量后好多了，但是还是比1.2.2版本慢

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。里面有专人值班
